kernel/Cargo.toml

```diff
@@ -35,6 +35,7 @@ log = { workspace = true, features = ["max_level_info", "release_max_level_info"
 packit.workspace = true
 libtcgtpm = { workspace = true, optional = true }
 zerocopy.workspace = true
+sha2 = { workspace = true, features = ["force-soft"] }
```

Comment:
just for curiosity, why we need to enable feature?

Reply:
It is related to https://github.com/RustCrypto/hashes/issues/446 that affects custom targets. svsm is  x86_64-unknown-none

Reply:
what about adding a comment there, because I'm not sure if `force-soft` affects something else

Reply:
Done

Reply:
Found this as I'm running into the same problem.